File: Napakalaki Ruby/Cultist.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

class Cultist
  def initialize(n,glevels)
    @name = n
    @gainedLevels = glevels
  end
  
  def getGainedLevels
    return @gainedLevels
  end
end

File: Napakalaki Java/NapakalakiGame/Cultist.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package NapakalakiGame;

/**
 *
 * @author Germán
 */
public class Cultist {
    
    private String name = null;
    private int gainedLevels = 0;
    
    public Cultist (String n, int l) {
        name = n;
        gainedLevels = l;
    }
    
    public int getGainedLevels() {
        return gainedLevels;
    }
}
